PoC - Vite and Tanstack Router, Query and `@plone/client` in SSR mode. (#5750)
plone/volto: apps/vite-ssr/README.md

# Plone on Vite with SSR mode

> This is a proof of concept of a [Vite](https://https://vitejs.dev) build, using `@plone/client` and `@plone/components` library.

Reviewer: ```suggestion
This is a proof of concept of a [Vite](https://vitejs.dev) build, using `@plone/client` and `@plone/components` libraries.
```

> This is intended to serve as both a playground for the development of both packages and as demo of Plone using Vite build with Server Side Rendering (SSR).

Reviewer: ```suggestion
This is intended to serve as both a playground for the development of both packages and as a demo of Plone using Vite built with server side rendering (SSR).
```

> It also uses TanStack Router as routing library.

Reviewer: ```suggestion
It also uses [TanStack Router](https://tanstack.com/router/latest/docs/framework/react/overview) for its routing library.
```
